### PR TITLE
Contribution templates

### LINKED ISSUE #2: Contribution
**Describe the problem you're trying to solve**
The goal is to elaborate a git workflow for contributing.

**Describe your solution**
Write templates for Issues & PR as well as a CONTRIBUTING.md in order to guide people through the convention they need to respect in order to contribute to the project.

**Describe the task list**
- [x] CONTRIBUTING.md
- [x] Issue & PR templates

### PR DESCRIPTION
**Pull request by Roxxas96**
## Related Issue
Closes #2 

## Overall description of your work
Created a CONTRIBUTING.md to guide people through the conventions to respect in order to contribute to the project & added some templates for Issues & PRs to respect in order to have a clear workflow based on feature branches.

## Task list
- [X] CONTRIBUTING.md
  - [X] Commits
  - [X] Issues
  - [X] Pull Requests
  - [X] Reviews
- [X] Issue & PR templates
  - [X] Bug report Issues
  - [X] Feature request Issues
  - [X] PR template

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+# Web BDE : Contributing guide
+
+This document aims at describing the flow of all sub-projects concerning Web BDE.
+
+## Git workflow
+
+The **main** repository [web-bde](https://github.com/Web-BDE/web-bde) is composed of **submodules**, each submodules concern a particular **micro-service** ([front-web-bde](https://github.com/Web-BDE/front-web-bde) for example is the service that distribute static files). Each micro service project has its own Issues/PR and follow **this workflow**.
+
+In each repository yo wil find 3 types of branch:
```
Comment (RedbeanGit) [inline CONTRIBUTING.md]:
```suggestion
In each repository you will find 3 types of branch:
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+The title of your Issue should describe the topic.
+
+You **must** respect the template provided, tow templates are provided : one for Feature/Documentation request, one for Bug report.
+
+Every Issues must contain a task list to follow the progress of the Issue :
+
+- [x] Write the contributing.md
+
+## When an how to make a Pull Request ?
```
Comment (RedbeanGit) [inline CONTRIBUTING.md]:
```suggestion
## When and how to make a Pull Request ?
```

**Review by RedbeanGit — APPROVED** [review]
Except for a few typos, everything looks good to me

File: CONTRIBUTING.md (hunk 1 above)
Comment (Roxxas96) [reply CONTRIBUTING.md]: Thank you for the typo !

File: CONTRIBUTING.md (hunk 2 above)
Comment (Roxxas96) [reply CONTRIBUTING.md]: Thank you !

**Comment by Roxxas96** [comment]
[Refacto]

I changed some descriptions about the branches type, added a section for fixup commits and submodules.
I also squashed fixup commis so the PR is ready to merge.
If you want to review the end file @RedbeanGit you can until tomorrow when I'll merge the branch.

**Review by Roxxas96 — COMMENTED** [review]
[Review]

I changed some descriptions about the branches type, added a section for fixup commits and submodules.
I also squashed fixup commis so the PR is ready to merge.
If you want to review the end file @RedbeanGit you can until tomorrow when I'll merge the branch.

**Pull request merged**